File: wkendtask/app/src/main/java/com/example/ahmedpc/wkendtask/MainActivity.java
package com.example.ahmedpc.wkendtask;

import android.content.DialogInterface;
import android.provider.Settings;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.ImageView;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    int count = 1;
    int total = 28;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);


        TextView res = findViewById(R.id.tv_res);
        ImageView foodie = findViewById(R.id.iv_image);
        TextView name = findViewById(R.id.tv_name);
        final Button price = findViewById(R.id.btn_price);
        TextView ingredient = findViewById(R.id.tv_ingredients);
        TextView quantity = findViewById(R.id.tvquan);
        Button decrement = findViewById(R.id.btndec);
        final TextView tv1 = findViewById(R.id.tv_1);
        Button increment = findViewById(R.id.btninc);
        Button size = findViewById(R.id.btn_size);
        Button dish = findViewById(R.id.btn_dish);
        Button sauce = findViewById(R.id.btn_sauce);
        Button topping = findViewById(R.id.btn_topping);
        TextView add = findViewById(R.id.tv_add);
        Button need = findViewById(R.id.btn_need);
        Button item = findViewById(R.id.btn_item);

        tv1.setText(String.valueOf(count));

        decrement.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {

                tv1.setText(String.valueOf(count--));
                total = count * 28;
                price.setText(String.valueOf("SAR" + "" + total));
                if (count<=1) {
                    count = 1;
                }
            }
        });

        increment.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {

                tv1.setText(String.valueOf(count++));
                total = count * 28;
                price.setText(String.valueOf("SAR" + "" + total));
                if (count >=10) {
                    count = 10;
                }
            }
        });
    }
}

File: SimpleFragments/app/src/main/java/com/example/simplefragments/MainActivity.kt
package com.example.simplefragments

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

        override fun onCreate(savedInstanceState: Bundle?) {
            super.onCreate(savedInstanceState)
            setContentView(R.layout.activity_main)

            button1.setOnClickListener {
                val textFragment = TextFragment()
                val manager = supportFragmentManager
                val transaction = manager.beginTransaction()
                transaction.replace(R.id.fragment_container,textFragment)
                transaction.addToBackStack(null)
                transaction.commit()
            }


            button2.setOnClickListener {
                val imageFragment = ImageFragment()
                val manager = supportFragmentManager
                val transaction = manager.beginTransaction()
                transaction.replace(R.id.fragment_container,imageFragment)
                transaction.addToBackStack(null)
                transaction.commit()
            }
        }
}
